app/Http/Controllers/API/TorrentController.php

```diff
@@ -241,6 +278,70 @@ public function show($id)
         return new TorrentResource($torrent);
     }
 
+    /**
+     * Edit the specified resource.
+     *
+     * @param Request $request
+     * @param int     $id
+     *
+     * @return TorrentResource
+     */
+    public function edit(Request $request, $id)
+    {
+        $user = $request->user();
+        if ($user->group->is_owner) {
+            $torrent = Torrent::findOrFail($id);
+            if ($request->filled('name')) {
```

Comment:
@HDVinnie For these if statements I only want to set $torrent->x if $request->filled('x') with no else. I don't want to touch the data if the request is supplied.

I found this stackoverflow, which one should I adopt to? I know PHP7.4 will be required soon but in the meantime I don't want to break anything for others who haven't updated.
https://stackoverflow.com/a/36933323/12882632

Reply:
its fine how it is since these are only ifs and not if else.